themes/default/content/blog/pulumi-release-notes-74/index.md

```diff
@@ -0,0 +1,169 @@
+- Pulumi Service & Pulumi.com
+  - [Stack READMEs](#stack-readmes)
+  - [SAML/SCIM improvements](#samlscim-improvements)
+
+## Cloud Providers and Packages
+
+### New resources in our providers
+
+We shipped new versions of the AWS Native provider, Google Native provider and the Azure Native provider that added support for 412 new resources in the last month. 385 of those resources were Azure Native, 7 AWS Native and 20 were added to Google Native.
+
+## Pulumi CLI and core technologies
+
+### Pulumi YAML v0.5.2
+
+We released v0.5.2 of Pulumi YAML which included bug fixes, new functions, diagnostics and validation. Some specific improvements we made were:
+
+- Add errors when hanging invalid fields off of resources.
+  [#203](https://github.com/pulumi/pulumi-yaml/pull/203)
+
+- Add errors when hanging invalid fields off of resource options.
+  [#211](https://github.com/pulumi/pulumi-yaml/pull/211)
+
+- Add a type checker.
+  [#228](https://github.com/pulumi/pulumi-yaml/pull/228)
+
+- Add `Fn::FromBase64`
+  [#218](https://github.com/pulumi/pulumi-yaml/pull/218)
+
+- Add support for Fn::ReadFile, enabling Pulumi Service [Stack README]({{< relref "/blog/stack-readme" >}}) support.
+  [#217](https://github.com/pulumi/pulumi-yaml/pull/217)
+
+- Allow Fn::Join to take expressions as inputs, previously the second argument had to be a syntactical list.
+  [#241](https://github.com/pulumi/pulumi-yaml/pull/241)
+
+- Fix `pulumi convert` panicking on programs containing `Fn::ToJSON`,`Fn::Secret` and `Fn::Invoke` (with an empty arguments property). [#250](https://github.com/pulumi/pulumi-yaml/pull/250), [#260](https://github.com/pulumi/pulumi-yaml/pull/260), and [#262](https://github.com/pulumi/pulumi-yaml/pull/262) respectively.
+
+As always, please feel free to submit feature requests and bug reports to the [Pulumi YAML GitHub Repo](https://github.com/pulumi/pulumi-yaml). We love hearing feedback from users!
+
+### Add `PreviewDigest` for third party tools
+
+Third party tools can now use `PreviewDigest` to be able to ingest the preview JSON. The `PreviewDigest` type can now be references directly from other programs instead of having to copy and paste it.
+
+Learn more in the [make previewDigest into exported type PreviewDigest GitHub issue](https://github.com/pulumi/pulumi/issues/9851).
+
+### Add local Policy Packs to Automation API
+
+The Pulumi [Automation API]({{< relref "/docs/guides/automation-api" >}}) is a programmatic interface for running Pulumi programs without the Pulumi CLI. Conceptually, this can be thought of as encapsulating the functionality of the CLI (`pulumi up`, `pulumi preview`, `pulumi destroy`, `pulumi stack init`, and so on.) but with more flexibility. We have now added support for Pulumi [Policy Packs]({{< relref "docs/guides/crossguard/get-started/#creating-a-policy-pack" >}}) can now be run with Automation API by specifying `--policy-pack`.
```

Comment:
```suggestion
The Pulumi [Automation API]({{< relref "/docs/guides/automation-api" >}}) is a programmatic interface for running Pulumi programs without the Pulumi CLI. Conceptually, this can be thought of as encapsulating the functionality of the CLI (`pulumi up`, `pulumi preview`, `pulumi destroy`, `pulumi stack init`, and so on.) but with more flexibility. We have now added support for Pulumi [Policy Packs]({{< relref "/docs/guides/crossguard/get-started#creating-a-policy-pack" >}}) can now be run with Automation API by specifying `--policy-pack`.
```